### ISSUE TITLE
Mismatched Results

### ISSUE THREAD

**Issue by yuqiuyuqiu** [mczhuge/ICON#11]
I retrained your model under the default settings, and got the following results, which are inconsistent with the paper's results:

Method:ICON-S,Dataset:ECSSD||Smeasure:0.92; meanEm:0.949; wFmeasure:0.898; MAE:0.036; adpEm:0.947; maxEm:0.958; adpFm:0.907; meanFm:0.909; maxFm:0.934; fnr:0.053
Method:ICON-S,Dataset:PASCALS||Smeasure:0.833; meanEm:0.866; wFmeasure:0.773; MAE:0.085; adpEm:0.863; maxEm:0.882; adpFm:0.792; meanFm:0.798; maxFm:0.829; fnr:0.098
Method:ICON-S,Dataset:DUTS-TE||Smeasure:0.85; meanEm:0.882; wFmeasure:0.76; MAE:0.057; adpEm:0.867; maxEm:0.906; adpFm:0.761; meanFm:0.786; maxFm:0.834; fnr:0.08
Method:ICON-S,Dataset:HKU-IS||Smeasure:0.912; meanEm:0.947; wFmeasure:0.881; MAE:0.032; adpEm:0.948; maxEm:0.958; adpFm:0.887; meanFm:0.893; maxFm:0.923; fnr:0.063
Method:ICON-S,Dataset:DUT-OMRON||Smeasure:0.818; meanEm:0.844; wFmeasure:0.712; MAE:0.078; adpEm:0.838; maxEm:0.876; adpFm:0.72; meanFm:0.739; maxFm:0.786; fnr:0.093
Method:ICON-S,Dataset:SOD||Smeasure:0.806; meanEm:0.851; wFmeasure:0.768; MAE:0.098; adpEm:0.847; maxEm:0.864; adpFm:0.812; meanFm:0.819; maxFm:0.829; fnr:0.24

Can you help me to solve it? Thank you very much!

**Comment by mczhuge** [comment]
Please check following settings:
(1) install the environment properly
(2) learning rate should be the same with [train_icon.sh](https://github.com/mczhuge/ICON/blob/main/util/scripts/train_icon.sh)
(3) check this line: https://github.com/mczhuge/ICON/blob/main/main/train.py#120

**Comment by yuqiuyuqiu** [comment]
Thank you very much!
